docs/zkapps/advanced-snarkyjs/actions-and-reducer.mdx

```diff
@@ -99,9 +99,63 @@ this.reducer.getActions({ fromActionsHash?: Field, endActionsHash?: Field }): A[
 
 Right now, `getActions` is available for [testing with `LocalBlockchain`](/zkapps/how-to-test-a-zkapp).
 
+### Fetching Actions from an Archive Node
+
+Since Mina nodes do not store historical network information and actions are not kept on-chain, emitted actions are only preserved in the Archive Node. Consequently, a zkApp must fetch its previously emitted actions on the Mina network from an Archive Node.
+
+As the Archive Node is essentially a database, it does not offer a user-friendly API that SnarkyJS can readily utilize to fetch actions. To address this issue, a GraphQL server was developed to retrieve network information such as actions and actions from the Archive Node. SnarkyJS will use this GraphQL server to fetch actions from the Archive Node.
```

Comment:
I see this content is repeated in another topic, let's sync up and talk about how we can repurpose the archive node description in one place and then have that place come to these new (awesome, btw!) topics